src/Uno.UI/UI/Xaml/Controls/AnimatedVisualPlayer/IAnimatedVisualSource.cs

```diff
@@ -18,6 +18,6 @@ public partial interface IAnimatedVisualSource
 
 		Size Measure(Size availableSize);
 
-		Uri UriSource { get; set; }
+		Uri Uri { get; set; }
```

Comment:
@carldebilly @Robert-Louis All things considered, does Uri need to be public at all? Is it part of the official API? If not, we could move it to an internal interface and have all the other animated visuals implement it.

Reply:
Unless I'm looking at wrong thing, this interface doesn't track at all the winui IAndimatedVisualSource (https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.controls.ianimatedvisualsource?view=windows-app-sdk-1.3)

Reply:
@jeromelaban as @nickrandolph point the interface never tracked the WinUI interface. Even if it's have the same name and is public, it look more like a internal interface for Uno.